### DISCUSSION TITLE
How to restart EMQX on Kubernetes Pod?

### DISCUSSION THREAD

**Discussion by iamonlysaiful** [emqx/emqx#8112] [Q&A] [answered]
As I follow this [EMQX Blog](https://www.emqx.com/en/blog/emqx-server-ssl-tls-secure-connection-configuration-guide) to apply self-signed certificate on my EMQX broker hosted on Kubernetes pod. 

There is an instruction that said to restart EMQX,
![image](https://user-images.githubusercontent.com/25750962/171349465-b28af779-d1ec-47ac-a4f0-bf71c9bf14fd.png)

Please help me to do so on the Kubernetes pod. As I rollout restarted my pod all changes on ***emqx.conf*** has gone

**Comment by zmstone** [comment, marked as answer]
Hi 
Are you using an older version of EMQX?
etc dir was previously a docker volume, so changes were not committed.
https://github.com/emqx/emqx/blob/4b867e6536695a04030215671dde06d8ec1765ea/deploy/docker/Dockerfile#L52

Please consider switching to 4.4.4
